channels.nix

```
@@ -2,7 +2,8 @@ rec {
   channels = {
       # "Channel name" = {
       #   job = "project/jobset/jobname"; like https://hydra.nixos.org/job/<value>/latest-finished
-      #   current = true; # when adding a new version, mark the oldest as "not current"
+      #   # when adding a new version, mark the oldest as "not current". New releases should always be current.
```

Channels where `current = false` are marked `end-of-life` on https://status.nixos.org/ and alerting is disabled.
```suggestion
      #   # when adding a new version, mark the oldest as "not current". New releases should always be current.
      #   # Channels where `current = false` are marked `end-of-life` on https://status.nixos.org/ and alerting is disabled.
```